patches/chrome-browser-flag_descriptions.cc.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/chrome/browser/flag_descriptions.cc b/chrome/browser/flag_descriptions.cc
+index 62fa3e5cb73b777cef2e0272fe2d892f352f18df..b083f1ec5f5691d147705dc15ba27f09e2e4d8d3 100644
+--- a/chrome/browser/flag_descriptions.cc
++++ b/chrome/browser/flag_descriptions.cc
+@@ -2000,6 +2000,10 @@ const char kSystemKeyboardLockDescription[] =
+     "keyboard shortcuts and have the events routed directly to the website "
+     "when in fullscreen mode.";
+ 
++const char kBraveAdblockCosmeticFilteringName[] = "Enable cosmetic filtering";
```

Comment:
this and the header file below can be done with chromium_src overrides. See `chromium_src/content/public/common/url_constants.cc`